rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/PackageNameConverter.groovy

```diff
@@ -33,177 +33,16 @@ class PackageNameConverter {
     String arch
 
     public String qualifiedPackageName(PackageType packageType) {
-      if (version && release) {
-        "${name}${packageType.getVersionDelimiter()}${version}-${release}"
-      } else if (version) {
-        "${name}${packageType.getVersionDelimiter()}${version}"
+      if (version) {
+        "${name}${packageType.util.versionDelimiter}${version}${release ? "-$release" : ''}"
```

Comment:
Yeah, these are confusing. Maybe `packageManagerVersionSeparator` instead of `versionDelimiter`?